diff --git a/node_modules/@rspress/runtime/dist/hooks.d.ts b/node_modules/@rspress/runtime/dist/hooks.d.ts
index 8e898d8..2e4f113 100644
--- a/node_modules/@rspress/runtime/dist/hooks.d.ts
+++ b/node_modules/@rspress/runtime/dist/hooks.d.ts
@@ -19,7 +19,7 @@ export declare function usePageData(): PageData;
 export declare function useLang(): string;
 export declare function useVersion(): string;
 export declare function useDark(): boolean;
-export declare function useI18n<T = Record<string, Record<string, string>>>(): (key: keyof T) => any;
+export declare function useI18n<T = Record<string, Record<string, string>>>(): (key: keyof T) => string;
 export declare function useViewTransition(dom: ReactNode): string | number | bigint | boolean | import("react").ReactElement<unknown, string | import("react").JSXElementConstructor<any>> | Iterable<ReactNode> | import("react").ReactPortal | Promise<string | number | bigint | boolean | import("react").ReactPortal | import("react").ReactElement<unknown, string | import("react").JSXElementConstructor<any>> | Iterable<ReactNode> | null | undefined> | null | undefined;
 export declare function useWindowSize(initialWidth?: number, initialHeight?: number): {
     width: number;
