diff --git a/node_modules/@rspress/runtime/dist/hooks/useI18n.d.ts b/node_modules/@rspress/runtime/dist/hooks/useI18n.d.ts
index 44e6f20..e2575e0 100644
--- a/node_modules/@rspress/runtime/dist/hooks/useI18n.d.ts
+++ b/node_modules/@rspress/runtime/dist/hooks/useI18n.d.ts
@@ -1 +1 @@
-export declare function useI18n<T = Record<string, Record<string, string>>>(): (key: keyof T) => any;
+export declare function useI18n<T = Record<string, Record<string, string>>>(): (key: keyof T) => string;
